src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -162,9 +164,9 @@ public GitSCM(
             List<BranchSpec> branches,
             Boolean doGenerateSubmoduleConfigurations,
             Collection<SubmoduleConfig> submoduleCfg,
-            GitRepositoryBrowser browser,
-            String gitTool,
-            List<GitSCMExtension> extensions) {
+            @CheckForNull GitRepositoryBrowser browser,
+            @CheckForNull String gitTool,
+            @NonNull List<GitSCMExtension> extensions) {
```

Comment:
Yes, there was no change in https://github.com/jenkinsci/git-plugin/pull/350
I just decided to ensure that there is no null value misusages before implementing the default behavior